### ISSUE TITLE
Admin pages failing

### ISSUE THREAD

**Issue by aaronoppenheimer** [Gig-o-Matic/GO3#454]
When I try to examine a gig I get an "Internal Server Error." Doesn't happen in local environment.

**Comment by aaronoppenheimer** [comment]
Neat. The log from digitalocean looks like this:

```
[web] [2024-04-24 00:01:55] [2024-04-23 20:01:55 -0400] [22] [ERROR] Error handling request /admin/gig/gig/894/change/?_changelist_filters=q%3Dtin%2Bhorn
[web] [2024-04-24 00:01:55] Traceback (most recent call last):
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 135, in handle
[web] [2024-04-24 00:01:55]     self.handle_request(listener, req, client, addr)
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/gunicorn/workers/sync.py", line 178, in handle_request
[web] [2024-04-24 00:01:55]     respiter = self.wsgi(environ, resp.start_response)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/wsgi.py", line 124, in __call__
[web] [2024-04-24 00:01:55]     response = self.get_response(request)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/base.py", line 140, in get_response
[web] [2024-04-24 00:01:55]     response = self._middleware_chain(request)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[web] [2024-04-24 00:01:55]     response = get_response(request)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/utils/deprecation.py", line 134, in __call__
[web] [2024-04-24 00:01:55]     response = response or self.get_response(request)
[web] [2024-04-24 00:01:55]                            ^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[web] [2024-04-24 00:01:55]     response = get_response(request)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/whitenoise/middleware.py", line 124, in __call__
[web] [2024-04-24 00:01:55]     return self.get_response(request)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[web] [2024-04-24 00:01:55]     response = get_response(request)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/utils/deprecation.py", line 134, in __call__
[web] [2024-04-24 00:01:55]     response = response or self.get_response(request)
[web] [2024-04-24 00:01:55]                            ^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[web] [2024-04-24 00:01:55]     response = get_response(request)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/utils/deprecation.py", line 134, in __call__
[web] [2024-04-24 00:01:55]     response = response or self.get_response(request)
[web] [2024-04-24 00:01:55]                            ^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[web] [2024-04-24 00:01:55]     response = get_response(request)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/utils/deprecation.py", line 134, in __call__
[web] [2024-04-24 00:01:55]     response = response or self.get_response(request)
[web] [2024-04-24 00:01:55]                            ^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[web] [2024-04-24 00:01:55]     response = get_response(request)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/utils/deprecation.py", line 134, in __call__
[web] [2024-04-24 00:01:55]     response = response or self.get_response(request)
[web] [2024-04-24 00:01:55]                            ^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[web] [2024-04-24 00:01:55]     response = get_response(request)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/utils/deprecation.py", line 134, in __call__
[web] [2024-04-24 00:01:55]     response = response or self.get_response(request)
[web] [2024-04-24 00:01:55]                            ^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[web] [2024-04-24 00:01:55]     response = get_response(request)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/utils/deprecation.py", line 134, in __call__
[web] [2024-04-24 00:01:55]     response = response or self.get_response(request)
[web] [2024-04-24 00:01:55]                            ^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[web] [2024-04-24 00:01:55]     response = get_response(request)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/utils/deprecation.py", line 134, in __call__
[web] [2024-04-24 00:01:55]     response = response or self.get_response(request)
[web] [2024-04-24 00:01:55]                            ^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[web] [2024-04-24 00:01:55]     response = get_response(request)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/utils/deprecation.py", line 134, in __call__
[web] [2024-04-24 00:01:55]     response = response or self.get_response(request)
[web] [2024-04-24 00:01:55]                            ^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[web] [2024-04-24 00:01:55]     response = get_response(request)
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/base.py", line 220, in _get_response
[web] [2024-04-24 00:01:55]     response = response.render()
[web] [2024-04-24 00:01:55]                ^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/response.py", line 114, in render
[web] [2024-04-24 00:01:55]     self.content = self.rendered_content
[web] [2024-04-24 00:01:55]                    ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/response.py", line 92, in rendered_content
[web] [2024-04-24 00:01:55]     return template.render(context, self._request)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/backends/django.py", line 61, in render
[web] [2024-04-24 00:01:55]     return self.template.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 175, in render
[web] [2024-04-24 00:01:55]     return self._render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
[web] [2024-04-24 00:01:55]     return self.nodelist.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
[web] [2024-04-24 00:01:55]     return SafeString("".join([node.render_annotated(context) for node in self]))
[web] [2024-04-24 00:01:55]                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/loader_tags.py", line 157, in render
[web] [2024-04-24 00:01:55]     return compiled_parent._render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
[web] [2024-04-24 00:01:55]     return self.nodelist.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
[web] [2024-04-24 00:01:55]     return SafeString("".join([node.render_annotated(context) for node in self]))
[web] [2024-04-24 00:01:55]                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/loader_tags.py", line 157, in render
[web] [2024-04-24 00:01:55]     return compiled_parent._render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
[web] [2024-04-24 00:01:55]     return self.nodelist.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
[web] [2024-04-24 00:01:55]     return SafeString("".join([node.render_annotated(context) for node in self]))
[web] [2024-04-24 00:01:55]                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/loader_tags.py", line 157, in render
[web] [2024-04-24 00:01:55]     return compiled_parent._render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
[web] [2024-04-24 00:01:55]     return self.nodelist.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
[web] [2024-04-24 00:01:55]     return SafeString("".join([node.render_annotated(context) for node in self]))
[web] [2024-04-24 00:01:55]                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/loader_tags.py", line 63, in render
[web] [2024-04-24 00:01:55]     result = block.nodelist.render(context)
[web] [2024-04-24 00:01:55]              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
[web] [2024-04-24 00:01:55]     return SafeString("".join([node.render_annotated(context) for node in self]))
[web] [2024-04-24 00:01:55]                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/loader_tags.py", line 63, in render
[web] [2024-04-24 00:01:55]     result = block.nodelist.render(context)
[web] [2024-04-24 00:01:55]              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
[web] [2024-04-24 00:01:55]     return SafeString("".join([node.render_annotated(context) for node in self]))
[web] [2024-04-24 00:01:55]                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/defaulttags.py", line 238, in render
[web] [2024-04-24 00:01:55]     nodelist.append(node.render_annotated(context))
[web] [2024-04-24 00:01:55]                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/loader_tags.py", line 208, in render
[web] [2024-04-24 00:01:55]     return template.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 177, in render
[web] [2024-04-24 00:01:55]     return self._render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
[web] [2024-04-24 00:01:55]     return self.nodelist.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
[web] [2024-04-24 00:01:55]     return SafeString("".join([node.render_annotated(context) for node in self]))
[web] [2024-04-24 00:01:55]                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/defaulttags.py", line 238, in render
[web] [2024-04-24 00:01:55]     nodelist.append(node.render_annotated(context))
[web] [2024-04-24 00:01:55]                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/defaulttags.py", line 238, in render
[web] [2024-04-24 00:01:55]     nodelist.append(node.render_annotated(context))
[web] [2024-04-24 00:01:55]                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/defaulttags.py", line 321, in render
[web] [2024-04-24 00:01:55]     return nodelist.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
[web] [2024-04-24 00:01:55]     return SafeString("".join([node.render_annotated(context) for node in self]))
[web] [2024-04-24 00:01:55]                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/defaulttags.py", line 321, in render
[web] [2024-04-24 00:01:55]     return nodelist.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
[web] [2024-04-24 00:01:55]     return SafeString("".join([node.render_annotated(context) for node in self]))
[web] [2024-04-24 00:01:55]                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 1070, in render
[web] [2024-04-24 00:01:55]     return render_value_in_context(output, context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 1047, in render_value_in_context
[web] [2024-04-24 00:01:55]     value = str(value)
[web] [2024-04-24 00:01:55]             ^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/utils/html.py", line 420, in <lambda>
[web] [2024-04-24 00:01:55]     klass.__str__ = lambda self: mark_safe(klass_str(self))
[web] [2024-04-24 00:01:55]                                            ^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/forms/boundfield.py", line 34, in __str__
[web] [2024-04-24 00:01:55]     return self.as_widget()
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/forms/boundfield.py", line 107, in as_widget
[web] [2024-04-24 00:01:55]     return widget.render(
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/forms/widgets.py", line 280, in render
[web] [2024-04-24 00:01:55]     context = self.get_context(name, value, attrs)
[web] [2024-04-24 00:01:55]               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/contrib/admin/widgets.py", line 320, in get_context
[web] [2024-04-24 00:01:55]     "rendered_widget": self.widget.render(name, value, attrs),
[web] [2024-04-24 00:01:55]                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/forms/widgets.py", line 281, in render
[web] [2024-04-24 00:01:55]     return self._render(self.template_name, context, renderer)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/forms/widgets.py", line 286, in _render
[web] [2024-04-24 00:01:55]     return mark_safe(renderer.render(template_name, context))
[web] [2024-04-24 00:01:55]                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/forms/renderers.py", line 29, in render
[web] [2024-04-24 00:01:55]     return template.render(context, request=request).strip()
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/backends/django.py", line 61, in render
[web] [2024-04-24 00:01:55]     return self.template.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 175, in render
[web] [2024-04-24 00:01:55]     return self._render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
[web] [2024-04-24 00:01:55]     return self.nodelist.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
[web] [2024-04-24 00:01:55]     return SafeString("".join([node.render_annotated(context) for node in self]))
[web] [2024-04-24 00:01:55]                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/defaulttags.py", line 238, in render
[web] [2024-04-24 00:01:55]     nodelist.append(node.render_annotated(context))
[web] [2024-04-24 00:01:55]                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/defaulttags.py", line 238, in render
[web] [2024-04-24 00:01:55]     nodelist.append(node.render_annotated(context))
[web] [2024-04-24 00:01:55]                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 966, in render_annotated
[web] [2024-04-24 00:01:55]     return self.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/loader_tags.py", line 208, in render
[web] [2024-04-24 00:01:55]     return template.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 177, in render
[web] [2024-04-24 00:01:55]     return self._render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 167, in _render
[web] [2024-04-24 00:01:55]     return self.nodelist.render(context)
[web] [2024-04-24 00:01:55]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/django/template/base.py", line 1005, in render
[web] [2024-04-24 00:01:55]     return SafeString("".join([node.render_annotated(context) for node in self]))
[web] [2024-04-24 00:01:55]                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[web] [2024-04-24 00:01:55]   File "/usr/local/lib/python3.12/site-packages/gunicorn/workers/base.py", line 203, in handle_abort
[web] [2024-04-24 00:01:55]     sys.exit(1)
[web] [2024-04-24 00:01:55] SystemExit: 1
[web] [2024-04-24 00:01:55] [2024-04-23 20:01:55 -0400] [22] [INFO] Worker exiting (pid: 22)
[web] [2024-04-24 00:01:56] [2024-04-24 00:01:56 +0000] [1] [ERROR] Worker (pid:22) was sent SIGKILL! Perhaps out of memory?
```

Looks like some infinite loop that gets killed after a while. I have made some changes to the admin system lately but it's stuff like making the titlebar a color. Since there's a lot of "rendering" referenced in the above maybe it's to do with that?

**Comment by aaronoppenheimer** [comment]
addressed in #455
